nx/include/switch/crypto.h

```diff
@@ -0,0 +1,17 @@
+/**
```

Comment:
Is this header really necessary? I would just put all these includes in switch.h directly.